src/adapter/src/catalog/storage.rs

```diff
@@ -466,7 +466,7 @@ async fn migrate(
                             .push(MzAclItem {
                                 grantee: role_id,
                                 grantor: MZ_SYSTEM_ROLE_ID,
-                                acl_mode: AclMode::CREATE,
```

Comment:
These changes come from Philip's comment: https://github.com/MaterializeInc/materialize/pull/19331#issuecomment-1554329235
@jkosh44 Are we interpreting that right that bootstrap user doesn't have enough permissions and it's correct to give them all and not just CREATE?